spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/client/loadbalancer/impl/CommonsLoadBalancer.java

```diff
@@ -0,0 +1,29 @@
+package org.springframework.cloud.client.loadbalancer.impl;
+
+import org.springframework.cloud.client.ServiceInstance;
+import org.springframework.cloud.client.loadbalancer.LoadBalancer;
+import org.springframework.cloud.client.loadbalancer.impl.scoped.ScopedLoadBalancer;
+import org.springframework.cloud.client.loadbalancer.impl.support.LoadBalancerClientFactory;
+
+/**
+ * @author Spencer Gibb
+ */
+public class CommonsLoadBalancer implements LoadBalancer {
```

Comment:
why Commons? Maybe Default? unless we are anticipating a tragedy of the commons where users eventually spoil the loadbalancer by using it without regards to others :)